.github/workflows/foundry-gas-diff.yml

```diff
@@ -0,0 +1,61 @@
+name: Report gas diff
+
+env:
+  RPC_URL_MAINNET: ${{ secrets.RPC_URL_MAINNET }}
+  RPC_URL_GOERLI: ${{ secrets.RPC_URL_GOERLI }}
+  RPC_URL_OPTIMISM: ${{ secrets.RPC_URL_OPTIMISM }}
+  RPC_URL_ARBITRUM: ${{ secrets.RPC_URL_ARBITRUM }}
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    # Optionally configure to run only for changes in specific files. For example:
+    # paths:
+    # - src/**
+    # - test/**
+    # - foundry.toml
+    # - remappings.txt
+    # - .github/workflows/foundry-gas-diff.yml
+
+jobs:
+  compare_gas_reports:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: 'core'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      # Add any step generating a gas report to a temporary file named gasreport.ansi. For example:
+      - name: Run tests
+        run: ./snapshot.sh > ../gasreport.ansi # <- this file name should be unique in your repository!
```

Comment:
Instead of trying to move to gasreport.ansi (which I'm not sure this will accomplish, since the script doesn't have any stdout)

Simply set the `report` in the Rubilmax action (https://github.com/Rubilmax/foundry-gas-diff#report-string)

Reply:
That doesn't work since the path name can't have a forward slash in it 🌵